time gpstime
float64 latitude
float64 longitude
float64 altitude
bool isAutoflight
int8 mode

